forms-flow-web/src/components/Modeller/ModelEditorHook.js

```diff
@@ -296,11 +296,7 @@ const EditModel = React.memo(
           <Button onClick={exportDiagram}>
             Deploy
           </Button>
-          {/*
-            TODO: Implement multi-tenancy
-            {MULTITENANCY_ENABLED ? <label className="deploy-checkbox"><input type="checkbox" />  public / multi-tenancy</label> : null}
-          */}
-          <label className="deploy-checkbox"><input type="checkbox" />  public / multi-tenancy</label>
+          {MULTITENANCY_ENABLED ? <label className="deploy-checkbox"><input type="checkbox" /> Deploy the workflow as public  <div data-bs-toggle="tooltip" data-bs-placement="top" title="If not  checked only deployed to this particular tenant else public to all" className="coursor-pointer btn text-primary"><i className="fa fa-info-circle"></i></div></label> : null}
```

Comment:
Action for the checkbox is not added. Also for tenant need to send tenantId in request

Reply:
sure